src/main/java/com/phraktle/histodiff/HistoDiff.java

```diff
@@ -20,18 +24,37 @@
     public static void main(String[] args) throws IOException {
         if (args.length < 2) {
             System.out
-                    .println("Usage: HistoDiff file1 file2 [sortBy] [threshold]");
+                    .println("Usage: HistoDiff file1/url1 file2/url2 [sortBy] [threshold]");
             System.exit(1);
         }
-        File file1 = new File(args[0]);
-        File file2 = new File(args[1]);
+        File file1 = parseFileArg(args, 0);
+        File file2 = parseFileArg(args, 1);
         int sortBy = parseArg(args, 2);
         int threshold = parseArg(args, 3);
 
         dump(sort(sortBy,
                 filter(sortBy, threshold, diff(parse(file1), parse(file2)))));
     }
 
+    static File parseFileArg(String[] args, int idx) {
+        if (args[idx].startsWith("http")) {
```

Comment:
A filename may also start with `http`. Make this more robust (e.g. include `://`).